nonprod-argocd-config/apps/envs/uat/valuesFile/values-uat-validator-01.yaml

```diff
@@ -81,6 +81,7 @@ config:
     db:
       useinmemory: false
       sqlitepath: ""
+      historicalTxCount: 5107067
```

Comment:
and historicalTxCount need to be added as env var  in ten-node helm charts too ?
eg:  db.useinmemory

            - name: HOST_DB_USEINMEMORY
              value: {{ .Values.config.host.db.useinmemory | default "false" | quote }}
            - name: HOST_DB_SQLITEPATH
              value: {{ .Values.config.host.db.sqlitepath | default "" | quote }}